### ISSUE TITLE
Is it a clerical error?

### ISSUE THREAD

**Issue by BaiFeng666** [erikbolumburu11/AssaultCubeInternal#1]
`
float X; //0x0004
float Y; //0x000C
float Z; //0x0008
`

Maby Y is 0x008 and Z is 0x00C
I am learning internal cheat, a lot of things are inexperienced

**Comment by erikbolumburu11** [comment]
Yes it's most likely a clerical error actually. Looks like I reordered the lines and forgot to change the comment.
